src/ontodia/diagram/linkView.ts

```diff
@@ -27,6 +29,19 @@ export class LinkView extends joint.dia.LinkView {
     getTypeModel(): FatLinkType {
         return this.view.model.getLinkType(this.model.get('typeId'));
     }
+
+    private onRoutingUpdate (_bendingPoint: { x: number, y: number }, properties?: { silent: boolean }) {
```

Comment:
Extra space after method name. Also, don't use underscore (`_`) in parameter names.